webapp/frontend/src/app/shared/device-hours.pipe.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { DeviceHoursPipe } from './device-hours.pipe';
+
+
+describe('DeviceHoursPipe', () => {
+  it('create an instance', () => {
+    const pipe = new DeviceHoursPipe();
+    expect(pipe).toBeTruthy();
+  });
+});
```

Comment:
one last nit. Can you add some tests to show how this pipe behaves when called with:

- 'humanize'
- 'device_hours
- null
- a invalid string

thanks! @bauzer714  this is great work